### PR TITLE
Create sleekflow-cohort.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/sleekflow-cohort.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Send Cohorts to Sleekflow
+description: Sync cohorts from Amplitude to Sleekflow
+status: new
+---
+
+!!!beta
+
+    This integration is currently in beta and is in active development. This integration is maintained by Master Concept Group. Contact the Master Concept Team by emailing <digital_lab@hkmci.com> for support with this integration.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:9]:
```suggestion
    This integration is in beta and is in active development. The integration is maintained by Master Concept Group. For support or questions about the integration, contact the Master Concept team directly at [digital_lab@hkmci.com](mailto:digital_lab@hkmci.com).
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+title: Send Cohorts to Sleekflow
+description: Sync cohorts from Amplitude to Sleekflow
+status: new
+---
+
+!!!beta
+
+    This integration is currently in beta and is in active development. This integration is maintained by Master Concept Group. Contact the Master Concept Team by emailing <digital_lab@hkmci.com> for support with this integration. 
+
+[Sleekflow](https://sleekflow.io/) is an Omni-Channel Social Engagement Platform that helps companies and users manage communication channels such as WhatsApp, Facebook Messenger, WeChat, Line, and Instagram. Sleekflow handles both inbound messaging management as well as outbound messaging campaigns. This integration lets you sync cohorts from Amplitude to Sleekflow.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:11]:
```suggestion
[Sleekflow](https://sleekflow.io/) is an Omni-Channel Social Engagement Platform that helps companies and users manage communication channels like WhatsApp, Facebook Messenger, WeChat, Line, and Instagram. Sleekflow handles both inbound messaging management as well as outbound messaging campaigns. This integration lets you sync cohorts from Amplitude to Sleekflow. 
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+!!!beta
+
+    This integration is currently in beta and is in active development. This integration is maintained by Master Concept Group. Contact the Master Concept Team by emailing <digital_lab@hkmci.com> for support with this integration. 
+
+[Sleekflow](https://sleekflow.io/) is an Omni-Channel Social Engagement Platform that helps companies and users manage communication channels such as WhatsApp, Facebook Messenger, WeChat, Line, and Instagram. Sleekflow handles both inbound messaging management as well as outbound messaging campaigns. This integration lets you sync cohorts from Amplitude to Sleekflow. 
+
+## Considerations
+
+- This integration is only available by request. Email <digital_lab@hkmci.com> to request access.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:15]:
```suggestion
- This integration is available by request. Email [digital_lab@hkmci.com](mailto:digital_lab@hkmci.com) to request access.
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- You must enable this integration in each Amplitude project you want to use it in.
+- You must have a paid Sleekflow plan to enable this integration.
+- Sleekflow requires at least an email address or a phone number as the unique contact identifier. The phone number needs to have a country code (the ‘+’ in front of the country code is optional). This means the User_ID or user property you select in Amplitude must contain an email address or phone number.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:18]:
```suggestion
- Sleekflow requires at least an email address or a phone number as the unique contact identifier. The phone number must include a country code (the `+` in front of the country code is optional). This means the `user_ID` or user property you select in Amplitude must contain an email address or phone number.
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+## Setup
+
+### Sleekflow setup
+
+1. Log into Sleekflow.
+2. Navigate to Channels > Integrations > More customizable extensions.
+3. Click "Connect" next to "API"
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:26]:
```suggestion
3. Click **Connect** next to **API**.
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- This integration is only available by request. Email <digital_lab@hkmci.com> to request access.
+- You must enable this integration in each Amplitude project you want to use it in.
+- You must have a paid Sleekflow plan to enable this integration.
+- Sleekflow requires at least an email address or a phone number as the unique contact identifier. The phone number needs to have a country code (the ‘+’ in front of the country code is optional). This means the User_ID or user property you select in Amplitude must contain an email address or phone number.
+
+## Setup
+
+### Sleekflow setup
+
+1. Log into Sleekflow.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:24]:
```suggestion
1. Log in to Sleekflow.
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+2. Navigate to Channels > Integrations > More customizable extensions.
+3. Click "Connect" next to "API"
+4. Copy the API Key to your clipboard.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:27]:
```suggestion
4. Copy the Sleekflow API Key.
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+### Amplitude setup
+
+1. In Amplitude Data, click **Catalog** and select the **Destinations** tab.
+2. In the Cohort section, click **Sleekflow**.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:32]: @Brandoncharleskhoo1 Is this integration available to everyone in Amplitude? If so, does the request to Sleekflow mentioned on line 15 enable the Sleekflow setup steps?

File: docs/data/destinations/sleekflow-cohort.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+3. Click **Add another destination**.
+4. Enter **Name** and paste in the **API** key you copied from **Sleekflow**.
+5. Map the **Amplitude fields** (which must contain an email address or phone number) to the **Sleekflow** fields.
+6. Save when finished.
+
+## Send a cohort
+
+To sync your first cohort, follow these steps:
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:40]:
```suggestion
To sync your first cohort:
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+1. In Amplitude, open the cohort you want to sync, then click **Sync**.
+2. Select **Sleekflow**, then click **Next**.
+3. Choose the account you want to sync to.
+4. Choose the sync cadence.
+5. When finished, save your work.
+
+### Use cases
+
+1. **Personalized Push Notifications:**  By integrating Amplitude cohorts with Sleekflow, you can send personalized messages to different user segments based on their behavior and preferences. For example, you can create cohorts of users who have abandoned their shopping carts and send them targeted reminders or incentives via WhatsApp, Facebook Messenger, or other communication channels supported by Sleekflow.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:50]:
```suggestion
1. **Personalized Push Notifications:**  When you integrate Amplitude cohorts with Sleekflow, you can send personalized messages to different user segments based on their behavior and preferences. For example, you can create a cohort of users who have abandoned their shopping carts then send them targeted reminders or incentives through WhatsApp, Facebook Messenger, or other communication channels that Sleekflow supports. 
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+2. **Re-Engagement Campaigns:** You can use Amplitude cohorts to identify users who haven't interacted with your brand for a certain period. Sleekflow can help you send re-engagement messages to these cohorts through various social media channels, encouraging them to revisit your platform and make a purchase.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:51]:
```suggestion
2. **Re-Engagement Campaigns:** Use Amplitude cohorts to identify users who haven't interacted with your brand for a certain period. Sleekflow can send re-engagement messages to these cohorts through supported social media channels, encouraging them to revisit your platform and make a purchase.
```

File: docs/data/destinations/sleekflow-cohort.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+3. **Product Recommendations:** Leverage Amplitude's user behavior data to create cohorts of users with similar product interests. Then, use Sleekflow to send product recommendations and promotions to these cohorts through channels like Instagram or Facebook Messenger, increasing the likelihood of conversions.
```
Comment (markzegarelli) [inline docs/data/destinations/sleekflow-cohort.md:52]:
```suggestion
3. **Product Recommendations:** Use Amplitude's user behavior data to create cohorts of users with similar product interests. Then, use Sleekflow to send product recommendations and promotions to these cohorts through channels like Instagram or Facebook Messenger, increasing the likelihood of conversions.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/data/destinations/sleekflow-cohort.md (hunk 8 above)
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/sleekflow-cohort.md:32]: Hi Mark, this integration is only available if Sleekflow approves it.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**